repl/evaluator.go

```diff
@@ -174,29 +202,37 @@ func (e *Evaluator) DelLetVar(name string) error {
 }
 
 func (e *Evaluator) applyContext() (*cel.Env, interpreter.Activation, error) {
-	env := e.env
 	var vars = make(map[string]interface{})
 	for _, el := range e.ctx.letVars {
+		if el.prog == nil {
```

Comment:
Could you add a comment about why you ignore the let vars that haven't been initialized?

Reply:
Done